src/docfx/Errors.cs

```diff
@@ -46,8 +46,11 @@ public static Error InvalidGitCommitsTime(string gitCommitsTimePath, Exception e
         public static Error DependenyRepoNotFound(string dependenyRepoHref)
             => new Error(ErrorLevel.Error, "dependency-repo-not-found", $"The dependency repository with href '{dependenyRepoHref}' is not found, make sure the `restore` command was executed");
 
-        public static Error AuthorNotFound(string author, DocfxException ex)
-            => new Error(ErrorLevel.Warning, "author-not-found", $"Author '{author}' cannot be recognized: {ex.Error.Message}");
+        public static Error AuthorNotFound(string author, DocfxInternalException ex)
```

Comment:
don't mix author not found and github rate limit exceeded into the same error code, they have different meanings and actions for end users.